src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import {IColumnFormatterProps, IColumnFormatterData, IColumnFormatter}
+    from "../../enhancedReactTable/IColumnFormatterProps";
+import "./mutationAssessor.scss";
+
+/**
+ * @author Selcuk Onur Sumer
+ */
+export default class MutationAssessorColumnFormatter extends React.Component<IColumnFormatterProps, {}> implements IColumnFormatter
+{
+    /**
+     * Mapping between the functional impact score (data) values and
+     * view values.
+     */
+    public static get MA_SCORE_MAP():{[key:string]:any} {
+        return {
+            h: {label: "High", style: "oma-high", priority: 4},
```

Comment:
typescript has enums.  would be good to have the style constants in an enum so they can be resused (e.g. down below)